go.mod

```diff
@@ -263,3 +263,5 @@ require (
 )
 
 replace github.com/tendermint/tendermint => github.com/bnb-chain/tendermint v0.31.12
+
+replace github.com/ledgerwatch/erigon-lib => github.com/roberto-bayardo/erigon-lib v0.0.0-20221220032111-191cf182196c
```

Comment:
testing for now?

Reply:
this is so it picks up the changes you just approved in those other libraries, since we're not yet submitting anything upstream.  so it will remain this way for a while.